webui/react/src/utils/types.ts

```diff
@@ -20,7 +21,7 @@ export const waitPageUrl = (command: Command): string | undefined => {
   if (!eventUrl || !proxyUrl) return;
   const event = encodeURIComponent(eventUrl);
   const jump = encodeURIComponent(proxyUrl);
-  return `/wait?event=${event}&jump=${jump}`;
+  return `${serverAddress(true)}/wait?event=${event}&jump=${jump}`;
```

Comment:
could this one line be all this ticket needs? if the Link elements gets an absolute path it wouldn't switch it for proxy.